CONTRIBUTING.md

```diff
@@ -0,0 +1,278 @@
+
+> **NOTE 1**: Image preprocessing parameters (mean and scale) must be built into a converted model to simplify model usage.
+
+> **NOTE 2**: If a model input is a color image, color channel order should be `BGR`.
+
+## Demo
+
+A demo shows the main idea of how to infer a model using IE. If your model solves one of the tasks supported by the Open Model Zoo, try to find an appropriate option from [demos](demos/README.md) or [samples](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_Samples_Overview.html). Otherwise, you must provide your own demo (C++ or Python). 
+
+The demo's name should end with `_demo` suffix to follow the convention of the project.
+
+Demos are required to support the following keys:
+
+ -  `-i "<input>"`: Required. Input to process.
+ -  `-m "<path>"`: Required. Path to an .xml file with a trained model. If the demo uses several models at the same time, use other keys prefixed with `-m_`.
+ -  `-d "<device>"`: Optional. Specifies a target device to infer on. CPU, GPU, FPGA, HDDL or MYRIAD is acceptable. Default must be CPU. If the demo uses several models at the same time, use keys prefixed with `d_` (just like keys `m_*` above) to specify device for each model.
+ -  `-no_show`: Optional. Do not visualize inference results.
+
+> **TIP**: For Python, it is preferable to use `--` instead of `-` for long keys.
+
+You can also add any other necessary parameters.
+
+If you add a new demo, provide autotesting support as well:
+- add demo launch parameters in [demos/tests/cases.py](demos/tests/cases.py)
+- prepare list of input images in [demos/tests/image_sequences.py](demos/tests/image_sequences.py)
+
+Add `reamde.md` file, which describes demo usage. Update [demos' README.md](demos/README.md) adding your demo to the list.
+
+## Accuracy Validation
+
+Accuracy validation can be performed by the [Accuracy Checker](./tools/accuracy_checker) tool. This tool can use either IE to run a converted model, or an original framework to run an original model. Accuracy Checker supports lots of datasets, metrics and preprocessing options, which simplifies validation if a task is supported by the tool. You only need to create a configuration file that contains necessary parameters for accuracy validation (specify a dataset and annotation, pre- and post-processing parameters, accuracy metrics to compute and so on). For details, refer to [Testing new models](./tools/accuracy_checker#testing-new-models).
```

Comment:
I think you need to mention that the config needs to be for the converted model.

Reply:
Thats why I left example below!

Reply:
A belated answer, but an example doesn't prove that the config needs to be for the converted model - it only says that it _can_ be.